R/WINDOWS.md

```diff
@@ -4,13 +4,21 @@ To build SparkR on Windows, the following steps are required
 
 1. Install R (>= 3.1) and [Rtools](http://cran.r-project.org/bin/windows/Rtools/). Make sure to
 include Rtools and R in `PATH`.
+
```

Comment:
This spaces are necessary because without them it will markdown as below:

<img width="818" alt="2016-08-31 3 56 32" src="https://cloud.githubusercontent.com/assets/6477701/18119268/cbbf8490-6f94-11e6-84e6-c9e875f00228.png">

This spaces make this pretty as below:

<img width="819" alt="2016-08-31 3 56 55" src="https://cloud.githubusercontent.com/assets/6477701/18119277/d48e0772-6f94-11e6-81ea-208533fc98e7.png">